MarvelCharsApp/MarvelCharsAppTests/SearchViewModelTests.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  SearchViewModelTests.swift
+//  MarvelCharsAppTests
+//
+//  Created by User on 11/30/21.
+//
+
+import XCTest
+@testable import MarvelCharsApp
+
+class SearchViewModelTests: XCTestCase {
+    var search = SearchViewModel()
+    var refreshCallsCount = 0
+    
+    
+    func prepareAndWaitFirstFetchExpectation() {
+        let expectationForFecthAll = self.expectation(description: "Request to API")
+        search.refreshData = {
+            self.refreshCallsCount += 1
+            expectationForFecthAll.fulfill()
+        }
+        search.getAllcharacters()
+        self.waitForExpectations(timeout: 2, handler: nil)
```

Comment:
unit tests must never use a real API, use the mocks here and remove the wait.

Reply:
Changed to mockAPI